samples/hosting/docker/Core_7/Receiver/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static async Task Main()
+    {
+        //required to prevent possible occurrence of .NET Core issue https://github.com/dotnet/coreclr/issues/12668
+        Thread.CurrentThread.CurrentUICulture = new CultureInfo("en-US");
+        Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
+
+        Console.CancelKeyPress += new ConsoleCancelEventHandler(OnExit);
+
+        Console.Title = "Samples.Docker.Receiver";
+
+        var endpointInstance = await TryStart()
+            .ConfigureAwait(false);
+        
+        Console.WriteLine("Press Ctrl+C to exit.");
+
+        // Wait until the message arrives.
+        closingEvent.WaitOne();
+
+        await endpointInstance.Stop()
+            .ConfigureAwait(false);
+    }
+
+    static async Task<IEndpointInstance> TryStart()
+    {
+        int count = 0;
+
+        while (count < 5)
+        {
+            ++count;
+
+            try
+            {
+                var endpointConfiguration = new EndpointConfiguration("Samples.Docker.Receiver");
+                endpointConfiguration.UsePersistence<InMemoryPersistence>();
+                endpointConfiguration.UseTransport<RabbitMQTransport>()
```

Comment:
You should add a call to `DelayedDelivery().DisableTimeoutManager()`. If you do that, you won't need to specify `InMemoryPersistence`.

Reply:
Thanks, will do!